docs/concepts/move.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Move"
+linkTitle: "Move"
+weight: 1
+---
+
+## Moving Ondat volume instances
+
+A volume instance (replica or primary) can be moved freely between nodes, this can be extremely useful in very specific scenarios, and it's available through a new command in the command-line tool.
+
+It allows to manually re-balance a cluster, reduce load on over-loaded nodes, etc.
+
+## Example of use
+
+Here's an example of how the CLI cmd works:
+
+```bash
+storageos update volume move my-volume-id source-node-id destination-node-id --namespace my-namespace-name
```

Comment:
What are <my_volume_id>, <source_node_id> etc? It's worth explaining these very explicitly. For example, volume ID could be: the PVC name, the PVC ID, the Ondat Volume UUID string etc. Same for the node ID.

Reply:
These are used in all the cli commands so I don't think we should explain them in every feature's page trying to account for this being the first page the user visits